src/main/java/org/pesmypetcare/webservice/service/FreqTrainingServiceImpl.java

```diff
@@ -0,0 +1,55 @@
+package org.pesmypetcare.webservice.service;
+
+
+import org.pesmypetcare.webservice.dao.FreqTrainingDao;
+import org.pesmypetcare.webservice.entity.FreqTrainingEntity;
+import org.pesmypetcare.webservice.error.DatabaseAccessException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Map;
+
+@Service
+public class FreqTrainingServiceImpl implements FreqTrainingService {
+    @Autowired
+    private FreqTrainingDao freqTrainingDao;
+
+
+    @Override
+    public void createFreqTraining(String owner, String petName, String date, FreqTrainingEntity freqTrainingEntity) {
+        freqTrainingDao.createFreqTraining(owner, petName, date, freqTrainingEntity);
+    }
+
+    @Override
+    public void deleteAllFreqTrainings(String owner, String petName) throws DatabaseAccessException {
+        freqTrainingDao.deleteAllFreqTrainings(owner, petName);
+    }
+
+    @Override
+    public void deleteFreqTrainingByDate(String owner, String petName, String petDate) throws DatabaseAccessException {
+        freqTrainingDao.deleteFreqTrainingByDate(owner, petName, petDate);
+    }
+
+    @Override
+    public FreqTrainingEntity getFreqTrainingByDate(String owner, String petName, String petDate) throws DatabaseAccessException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 130).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5400494)